src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -24,11 +24,13 @@ const NavBar = lazy(() =>
 export interface WithNavigationBarProps {
   hasBackButton?: boolean;
   withSchedule?: boolean;
+  withPhotobooth?: boolean;
 }
 
 export const WithNavigationBar: React.FC<WithNavigationBarProps> = ({
   hasBackButton,
   withSchedule = true,
+  withPhotobooth = true,
```

Comment:
Might be better to always have these boolean props default to `false`, that way it is a simplified use:

- `<WithNavigationBar>` when none are available,
- `<WithNavigationBar hasBackButton hasSchedule hasPhotobooth >` when all are available

Reply:
agree, redone this logic in b230401